catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -109,6 +113,17 @@ public void init() {
           res.header("Content-Encoding", "gzip");
         });
 
+    get(
+        "/cql/transforms",
+        (req, res) -> {
```

Comment:
✏️ Alternatively,
```
  (req, res) -> queryResponseTransformers
                .stream()
                .map(...)
                .collect(toList()),
  mapper::toJson
});
```

if you want to separate the endpoint logic from the response transformation.

Reply:
This is a placeholder action that is here just for the purpose of fulfilling all of the export control functionality and will need to be refactored.